packages/desktop/views/login/views/SelectProfileView.svelte

```diff
@@ -24,8 +24,7 @@
         const isDeveloperProfile = shouldBeDeveloperProfile()
         await initialiseOnboardingFlow({
             isDeveloperProfile,
-            networkProtocol: NetworkProtocol.Shimmer,
-            ...(!isDeveloperProfile && { networkType: NetworkType.Mainnet }),
+            ...(isDeveloperProfile && { networkProtocol: NetworkProtocol.Shimmer }),
```

Comment:
Is this supposed to be `NetworkId`? Also I'd think this is supposed to be `Testnet` and not `Shimmer` right?